tests/run/cyfunction_defaults.pyx

```diff
@@ -2,6 +2,8 @@
 # mode: run
 # tag: cyfunction, closures
 
+from libcpp.vector cimport vector
```

Comment:
This adds C++ usage to a test that does not depend on C++ compilation. It seems better to add a new test file `cyfunction_defaults_cpp.pyx` with a `cpp` tag for this test (and for similar ones, there's probably more that we could test here).